### PR TITLE
chore: clippy happy

### PR DESCRIPTION
**Pull request by mattsse**
```
error: lint group `rust_2018_idioms` has the same priority (0) as a lint
   --> Cargo.toml:109:6
    |
108 | rust.unused_must_use = "deny"
    |      --------------- has the same priority as this lint
109 | rust.rust_2018_idioms = "deny"
    |      ^^^^^^^^^^^^^^^^   ------ has an implicit priority of 0
    |
    = note: the order of the lints in the table is ignored by Cargo
    = help: for further information visit https://rust-lang.github.io/rust-clippy/master/index.html#lint_groups_priority
    = note: `#[deny(clippy::lint_groups_priority)]` on by default
help: to have lints override the group set `rust_2018_idioms` to a lower priority
    |
109 | rust.rust_2018_idioms = { level = "deny", priority = -1 }
```

### CODE REVIEW

**Review by shekhirin — APPROVED** [review]

**Comment by shekhirin** [comment]
fixing more issues

File: examples/node-event-hooks/src/main.rs [hunk 1]
```diff
@@ -10,10 +10,10 @@
 //! This launch the regular reth node and also print:
 //!
 //! > "All components initialized"
-//! once all components have been initialized and
+//! > once all components have been initialized and
```
Comment (DaniPopes) [inline examples/node-event-hooks/src/main.rs:13]: this is wrong

**Review by DaniPopes — CHANGES_REQUESTED** [review]

File: examples/node-event-hooks/src/main.rs (hunk 1 above)
Comment (mattsse) [reply examples/node-event-hooks/src/main.rs:13]: what is correct?

Comment (mattsse) [reply examples/node-event-hooks/src/main.rs:13]: I guess there should be a new line?
@shekhirin

**Pull request merged**